src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoRoot)\build\CodeAnalysis.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;net461;netstandard2.0;net5.0</TargetFrameworks>
+    <Description>OpenTelemetry .NET SDK Extensions</Description>
```

Comment:
I'd vote to keep this more specific rather than a general package Extensions. Like extensions.LogToActivityConverter..

Also, this conversion (and other conversions like ActivityEvent -> Logs, or Activity -> Metrics) could be made part of the SDK itself in the future. But to allow us more time to bake this feature, we can keep it separate package for now (and only do pre-releases,). When its considered good for 1.0 stable quality, we can revisit the applicability of the package and promote to the core SDK package itself.

Reply:
I initially thought of naming these otel.experimental to really warn users about making production level bet on this, but then realized that our versioning doc is already making it clear that any prerelease can contain breaking change or even feature removal.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/VERSIONING.md#pre-releases

Reply:
Ya @reyang was telling me about some of the spec conversations in this area that's why I moved it out of SDK. I'm good with whatever you want to do, but having an assembly just for 1 thing seems overkill to me. What about something like OpenTelemetry.Preview.Adapters? 🤷

Reply:
+1 on "but having an assembly just for 1 thing seems overkill to me"

Reply:
Okay to use Otel.Extensions as project name. (One qn is should we move to contrib repo or keep it here)

Reply:
Moved to contrib